policyengine_us/variables/gov/states/id/tax/income/deductions/itemized/id_itemized_deduction.py

```diff
@@ -0,0 +1,57 @@
+from policyengine_us.model_api import *
+import numpy as np
+
+
+class id_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.idaho.gov/wp-content/uploads/forms/EIS00407/EIS00407_01-05-2023.pdf"
+        "https://tax.idaho.gov/wp-content/uploads/forms/EIN00046/EIN00046_03-01-2023.pdf"
+    )
+    defined_for = StateCode.ID
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.id.tax.income.deductions.itemized
+        itm_deds = [
+            deduction
+            for deduction in p.amount
+            if deduction not in ["salt_deduction"]
+        ]
```

Comment:
dont need anything below this

Reply:
but in the reference there are more steps: 
2. get the standard deductions amount
3. Subtract line 2 (standard deductions) from line 1 (itemized deductions). If less than zero, enter”0” 
4. Divide line 3 by $3,417
the code above only stands for step 1 in the reference. 
so we don't need steps 2-4 here?

Reply:
which reference - please add link